templates/parachain/pallets/template/src/lib.rs

```diff
@@ -66,18 +66,19 @@ mod benchmarking;
 // To see a full list of `pallet` macros and their use cases, see:
 // <https://paritytech.github.io/polkadot-sdk/master/pallet_example_kitchensink/index.html>
 // <https://paritytech.github.io/polkadot-sdk/master/frame_support/pallet_macros/index.html>
-#[frame_support::pallet]
+#[polkadot_sdk::frame_support::pallet]
 pub mod pallet {
-	use frame_support::{dispatch::DispatchResultWithPostInfo, pallet_prelude::*, DefaultNoBound};
-	use frame_system::pallet_prelude::*;
-	use sp_runtime::traits::{CheckedAdd, One};
+	use polkadot_sdk::{
```

Comment:
No matter the broader conclusion, please take this small step here: Import `polkadot_sdk::polkadot_sdk_frame::prelude::*`. It will help you remove a bunch of unneeded stuff.

Reply:
Done. Used `frame` as a dependency for the pallet. Indeed made things easier.